config/locales/en/shared.yml

```diff
@@ -50,6 +50,10 @@ en:
         sub_heading: Select all that apply
         or_break: or
         none_selected: None of these
+      savings_and_investments_form:
+        sub_heading: Select all that apply
+        or_break: or
```

Comment:
Rather than duplicating `or_break: or` here and in two other translations, I think it would better to remove the local `or_break`, and use `generic.or` which has the same value, and can replace the usage of all these entries.